android/data/src/main/java/org/fossasia/openevent/data/Feedback.java

```diff
@@ -0,0 +1,37 @@
+package org.fossasia.openevent.data;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.github.jasminb.jsonapi.LongIdHandler;
+import com.github.jasminb.jsonapi.annotations.Id;
+import com.github.jasminb.jsonapi.annotations.Relationship;
+import com.github.jasminb.jsonapi.annotations.Type;
+
+import org.fossasia.openevent.core.auth.model.User;
+
+import io.realm.RealmObject;
+import io.realm.annotations.PrimaryKey;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+@Data
+@Builder
```

Comment:
Why builder?

Reply:
i thought we could use the same POJO with a little modification for posting feedback part that's why i kept builder

Reply:
Yeah maybe my mistake previously, but only use builder or other annotations when needed like in User or Event